uvm/hier/usr/lib/python3/dist-packages/tests/global_functions.py

```diff
@@ -594,6 +594,83 @@ def build_nmap_command(sudo=True, override_arguments=None, extra_arguments=None,
     print(f"{sys._getframe().f_code.co_name}: {command}" )
     return command
 
+def get_wait_for_command_result(command=None, success_result=0, tries=10, local=False):
+    """
+    Wait up to tries for a successful command result
+
+    returns True if ping successful, False otherwise
+    """
+    if command is None:
+        command = build_ping_command(target=TEST_SERVER_HOST)
+
+    result = None
+    while result != 0 and tries > 0:
+        tries -= 1
+        if local is True:
+            result = subprocess.run(command, shell=True, stderr=subprocess.PIPE, stdout=subprocess.PIPE)
+            result = result.returncode
+        else:
+            result = remote_control.run_command(command)
+
+        if result == success_result:
+            return True
+        time.sleep(1)
+
+    return False
+
+def get_wait_for_command_output(command=None, success_test=None, tries=10, local=False):
+    """
+    Wait up to tries for a successful command result
+
+    returns True if ping successful, False otherwise
+    """
+    if command is None:
+        command = build_ping_command(target=TEST_SERVER_HOST)
+
+    result = None
+    while result != 0 and tries > 0:
+        tries -= 1
+        if local is True:
+            result = subprocess.run(command, shell=True, stderr=subprocess.STDOUT, stdout=subprocess.PIPE)
+            result = result.stdout.decode('utf-8')
+        else:
+            result = remote_control.run_command(command, stdout=True)
+
+        result = result.strip()
+        print(f"result={result}")
+        if success_test is None or success_test(result):
+            return result
+        time.sleep(1)
+
+    return ""
```

Comment:
This will probably work fine since "" is also Falsy, but did you intend to return False here?

Reply:
This method is actually returning from stdout, so an empty string is what we need for a false value here